simple-jwt-login/src/Modules/WordPressDataInterface.php

```diff
@@ -286,4 +286,10 @@ public function getUserRoles($user);
      * @return bool
      */
     public function isUserLoggedIn();
+
+    /**
+     * @param string $value
+     * @return string
+     */
+    public function sanitizePassword(string $value): string;
```

Comment:
Please remove the return type, as the plugin can also be used in older PHP versions.

Reply:
Removed the return types! Sorry about that!